test/metabase/util/date_2_test.clj

```diff
@@ -586,3 +586,8 @@
   (testing "can handle infinity dates (#12761)"
     (is (u.date/with-time-zone-same-instant java.time.OffsetDateTime/MAX (t/zone-id "UTC")))
     (is (u.date/with-time-zone-same-instant java.time.OffsetDateTime/MIN (t/zone-id "UTC")))))
+
+(deftest standard-offset-test
+  (testing "standard-offset works correctly"
+    (is (= (t/zone-offset "-06:00") (u.date.common/standard-offset (t/zone-id "America/Chicago"))))
```

Comment:
You should add some tests that use `mt/with-clock` to switch the clock to something definitely in DST and something definitely not in DST to make sure it returns the same offset regardless of the system clock date